TF_Tooling_Web/src/Plugins/tests/InputActionManager.test.ts

```diff
@@ -0,0 +1,64 @@
+import { HandChirality, HandType, InputType, InteractionType, TouchFreeInputAction } from '../../TouchFreeToolingTypes';
+import { mockTfPluginInputAction } from '../../tests/testUtils';
+import { InputActionManager } from '../InputActionManager';
+import { InputActionPlugin } from '../InputActionPlugin';
+
+describe('InputActionManager', () => {
+    let currentInputAction: TouchFreeInputAction;
+    let currentModifiedAction: TouchFreeInputAction;
+    let pluginCalled = false;
+
+    class MockPlugin extends InputActionPlugin {
+        RunPlugin(inputAction: TouchFreeInputAction): TouchFreeInputAction | null {
+            pluginCalled = true;
```

Comment:
💬 Would we want to do this after the following expect checks? So we know they've been run through?

Reply:
Sure - will move it :)